mayalookassigner/app.py

```diff
@@ -109,8 +113,8 @@ def setup_ui(self):
 
         # Method buttons
         method_buttons_layout = QtWidgets.QHBoxLayout()
-        assign_to_selected_btn = QtWidgets.QPushButton("Assign to Selected")
-        assign_to_all_btn = QtWidgets.QPushButton("Assign to All")
+        assign_to_selected_btn = QtWidgets.QPushButton("Process Selected Queue")
```

Comment:
Actually, this label might be more confusing? It's not actually doing anything with the queue with this button? Correct?

It's only assigning the things of the top view (that is not the queue) to what is currently selected. Or am I misreading the code?